build.zig

```diff
@@ -1149,9 +1250,9 @@ const GitCloneStep = struct {
 /// - From MacOS, similar to https://github.com/ziglang/zig/issues/9711#issuecomment-1090071087.
 /// Workaround: Just setting different cache folders for each platform.
 fn builder_with_isolated_cache(
```

Comment:
After the PR lands, we need to check if we can remove this altogether, perhaps the underlying bugs were fixed?